.gitignore

```diff
@@ -60,6 +60,8 @@ yarn-error.log
 /ios/Pods/
 /vendor/bundle/
 /ios/Podfile.lock
+ios/Ts_Assignment.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
```

Comment:
No need to give a project-specific path! Check the above comments on how we can ignore them.